libs/application/template-api-modules/src/lib/modules/templates/directorate-of-immigration/citizenship/citizenship.service.ts

```diff
@@ -341,33 +350,33 @@ export class CitizenshipService extends BaseTemplateApiService {
           passportNumber: applicantPassport.passportNumber,
           passportTypeId: parseInt(applicantPassport.passportTypeId),
           countryOfIssuerId: applicantPassport.countryOfIssuerId,
-          file: await this.getFilesFromAttachment(
+          file: await this.getUrlForAttachment(
```

Comment:
If I'm understanding everything correctly, lots of things in the interface CitizenshipApplication are named something along the lines of: 
`filename: string; base64: string`
Which, after this change, wont actually be a base64 encoded string, but instead a url to the file. might want to look into renaming those fields? I don't know the greater scope of that change but it would avoid some confusion down the line potentially 😅